scripts/scripts.js

```diff
@@ -16,11 +16,44 @@ import {
 const PRODUCTION_DOMAINS = ['www.elixirsolutions.com'];
 const LCP_BLOCKS = []; // add your LCP blocks to the list
 
+function buildNewsColumns(main) {
+  if (!document.body.classList.contains('news')) {
+    return;
+  }
+
+  const h1 = main.querySelector('h1');
+  if (!h1) {
+    return;
+  }
+  const section = h1.closest('div');
+  const firstColElems = [];
+  const secondColElems = [];
+  let h1found = false;
+  [...section.children].forEach((elem) => {
```

Comment:
I'd try to use `.filter` to make the code easier to read:
```
const firstColElems = [...section.children].filter(elem => elem === h1);
```

Reply:
I don't think that will work.

the point of this code is that any element before the H1 goes in the first col, and anything after goes in the second col. so we are not simply finding the h1, rather, we are looking all the elements, and segmenting the into 2 arrays based on the h1 position.